CASMINST-5089 Add References to Manual Steps (#2106)
Cray-HPE/docs-csm: install/pre-installation.md

1. (`pit#`) Download the SHCD to the `prep` directory.

>     This will need to be retrieved from the administrators Cray deliverable.

Reviewer: ```suggestion
    This will need to be retrieved from the administrator's Cray deliverable.
```

1. Validate the SHCD.

    See [Validate SHCD](../operations/network/management_network/validate_shcd.md) and then return to this page.

### 3.2 Generate topology files

The following steps use the new, automated method for generating files. The previous step for

> [validate SHCD](#31-validate-shcd) will have generated "paddle" files, these are necessary for generating

Reviewer: ```suggestion
[validate SHCD](#31-validate-shcd) generated "paddle" files; these are necessary for generating
```

> the rest of the seed files.

Reviewer 2: Is this still true? The steps below are the old manual way.

Author: Needs rewording, the notice I added contradicts this.

> ***NOTE*** The paddle files are temporarily not used due to bugs in the seed file generation software.
> Until these bugs are resolved, the seed files must be manually generated.

> If seed files from a prior installation of the same major-minor version of CSM exist, these can be used and

Reviewer: ```suggestion
If seed files from a prior installation of the same major-minor version of CSM exist, then these can be used and
```

this step may be skipped.

> 1. (`pit#`) Create each seed file unless they already exist from a previous installation:

Reviewer: ```suggestion
1. (`pit#`) Create each seed file, unless they already exist from a previous installation.
```

   - For new installations of CSM that have no prior seed files, each one must be created:

>       - [Create `application_node_config.yaml`](./create_application_node_config_yaml.md)

Reviewer: ```suggestion
      - [Create `application_node_config.yaml`](create_application_node_config_yaml.md)
```

>       - [Create `cabinets.yaml`](./create_cabinets_yaml.md)

Reviewer: ```suggestion
      - [Create `cabinets.yaml`](create_cabinets_yaml.md)
```

>       - [Create `hmn_connections.json`](./create_hmn_connections_json.md)

Reviewer: ```suggestion
      - [Create `hmn_connections.json`](create_hmn_connections_json.md)
```

>       - [Create `ncn_metadata.csv`](./create_ncn_metadata_csv.md)

Reviewer: ```suggestion
      - [Create `ncn_metadata.csv`](create_ncn_metadata_csv.md)
```

>       - [Create `switch_metadata.csv`](./create_switch_metadata_csv.md)

Reviewer: ```suggestion
      - [Create `switch_metadata.csv`](create_switch_metadata_csv.md)
```

   - For re-installations of CSM 1.3, the previous seed files may be used and this step can be skipped.

>    - For new installations of CSM 1.3 that have prior seed files from CSM 1.2 or older, the following files

Reviewer: ```suggestion
   - For new installations of CSM 1.3 that have prior seed files from CSM 1.2 or older, the previous seed files
```

>    need to be recreated due to changes in their content or formatting:

Reviewer: ```suggestion
   may be used **except that the following files must be recreated** because of content or formatting changes:
```

>       - [Create `cabinets.yaml`](./create_cabinets_yaml.md)

Reviewer: ```suggestion
      - [Create `cabinets.yaml`](create_cabinets_yaml.md)
```

>       - [Create `hmn_connections.json`](./create_hmn_connections_json.md)

Reviewer: ```suggestion
      - [Create `hmn_connections.json`](create_hmn_connections_json.md)
```

> 

Reviewer: ```suggestion
```

>    The other files can be re-used (e.g. `application_node_config.yaml`, `ncn_metadata.csv`, and `switch_metadata.csv`).

Reviewer 2: This line I think need to be intended one level, so it appears underneath the bullet point correctly
![Screen Shot 2022-07-22 at 11 15 49 AM](https://user-images.githubusercontent.com/77114568/180481163-5b494941-b9ad-4541-a873-9371318c9552.png)
.

Author: Yup

>    The other files can be re-used (e.g. `application_node_config.yaml`, `ncn_metadata.csv`, and `switch_metadata.csv`).

Reviewer: ```suggestion
```

> 1. (`pit#`) Confirm that the following files exist:

Reviewer: ```suggestion
1. (`pit#`) Confirm that the following files exist.
```

   ```bash

>    ls -l ${PITDATA}/prep/{application_node_config.yaml,cabinets.yaml,hmn_connections.json,ncn_metadata.csv,switch_metadata.csv}

Reviewer: ```suggestion
   ls -l "${PITDATA}"/prep/{application_node_config.yaml,cabinets.yaml,hmn_connections.json,ncn_metadata.csv,switch_metadata.csv}
```

   ```

   Expected output may look like:

>    ```bash

Reviewer: ```suggestion
   ```text
```
